media/cc_color_scale.xml

```diff
@@ -14,11 +14,12 @@
             <step r="114" g="159" b="207" pos="0.020202020202"/>
             <step r="78" g="154" b="6" pos="0.030303030303"/>
             <step r="252" g="233" b="79" pos="0.040404040404"/>
-            <step r="239" g="41" b="41" pos="0.0505050505051"/>
-            <step r="173" g="127" b="168" pos="0.0606060606061"/>
-            <step r="85" g="87" b="83" pos="0.0707070707071"/>
-            <step r="178" g="102" b="255" pos="0.0808080808081"/>
-            <step r="255" g="0" b="127" pos="0.0909090909091"/>
+            <step r="239" g="41" b="41" pos="0.0505050505"/>
+            <step r="173" g="127" b="168" pos="0.0606060606"/>
+            <step r="85" g="87" b="83" pos="0.0707070707"/>
+            <step r="178" g="102" b="255" pos="0.0808080808"/>
+            <step r="255" g="0" b="127" pos="0.0909090909"/>
+            <step r="255" g="197" b="98" pos="0.09090909095"/>
```

Comment:
Moet die laatste dan niet zijn `pos="0.10101010101"` ofzoiets?

Reply:
Dat dacht ik ook, maar dan krijgt hij zelfde kleur als emmer?!

Reply:
Heeft met de `steps` setting in CloudCompare te maken denk ik, die moet op 100 staan.